hazelcast/src/test/java/com/hazelcast/wan/impl/WanNoDelayReplicationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.hazelcast.wan.impl;
+
+import com.hazelcast.map.impl.wan.MapReplicationUpdate;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.wan.ReplicationEventObject;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+
+/**
+ * @author Henri Tremblay
+ */
```

Comment:
Or better, there already is a test class related to wan replication -> [WanReplicationTest](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/java/com/hazelcast/wan/WanReplicationTest.java)
Could you please move your test cases to that class?